conntable.go

```diff
@@ -71,20 +74,42 @@ func (t *connTable) Register(ck ckey, matchedRule *Rule, srcIP, srcPort string,
 			TargetPort: targetPort,
 			//TargetIP:   targetIP,
 		}
+
 	}
 }
 
-func (t *connTable) FlushOlderThan(s time.Duration) {
+func (t *connTable) FlushOlderOnes() {
 	t.mtx.Lock()
 	defer t.mtx.Unlock()
 
-	threshold := time.Now().Add(-1 * s)
+	for minutes := 5; minutes >= 0; minutes -= 1 {
```

Comment:
pull `5` out into a constant. Or should this be a variable? Is there a use case for a user wanting to keep more than 1000 connections active that haven't received a packet in 5 minutes?